.github/workflows/ci.yml

```diff
@@ -318,10 +362,14 @@ jobs:
           STAGING_ACCESS_KEY_ID: ${{ secrets.STAGING_ACCESS_KEY_ID }}
           STAGING_SECRET_ACCESS_KEY: ${{ secrets.STAGING_SECRET_ACCESS_KEY }}
           GH_PAT: ${{ secrets.GH_PAT }}
-        run: |
-          export AWS_ACCESS_KEY_ID=$(echo "${ACCESS_KEY_IDS}" | jq -r ".[${ACCOUNT_INDEX}]")
-          export AWS_SECRET_ACCESS_KEY=$(echo "${SECRET_ACCESS_KEYS}" | jq -r ".[${ACCOUNT_INDEX}]")
-          yarn test $(echo $MODULES | jq -r '."${{matrix.module}}"')
+        with:
+          max_attempts: 2
+          retry_on: error
+          timeout_minutes: 60
```

Comment:
what's the difference between this timeout and the one in line 354?

Reply:
actually i will move to 90 minutes there. The retry action needs a timeout to be specified. But agree, needs to be the same